### PR TITLE
Add documentation.

### PR DESCRIPTION
**Pull request by samadpaydar3**
Add documentation to functions.

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -419,3 +455,20 @@ async function writeFilePart(part: Uint8Array, url: string): Promise<string> {
   } 
   return eTag;
 }
+
+/**
+ * Creates RepositoryApiClient fro cloud, from an access key.
```
Comment (jason-chan-lf) [inline index.ts]: *fro typo

File: index.ts [hunk 2]
```diff
@@ -156,6 +171,9 @@ async function importDocument(folderEntryId: number | undefined, sampleProjectFi
   return importedEntryId;
 }
 
+/**
+ * Sets fields on the entry specified by the given entry Id.
```
Comment (jason-chan-lf) [inline index.ts]: Maybe want to be more specific here like 
`Sets a string field on the entry specified by the given entry Id.`

Comment (samadpaydar3) [reply index.ts]: Fixed.

File: index.ts (hunk 1 above)
Comment (samadpaydar3) [reply index.ts]: Fixed.

**Review by jason-chan-lf — APPROVED** [review]

**Pull request merged**